Game_Manual/Stats.md

```diff
@@ -29,31 +29,83 @@ How much your team members felt that you supported them in their learning. This
+
+In the above dataset, the player's expected contribution for the `#big-bees` project is equal to 33.33% ( 100% / 3 ) because their team has 3 players, so an equal contribution is 1/3 of the project.
+
+### Contribution Gap
+
+The contribution gap is the difference between actual and expected contribution. It demonstrates whether a player contributed _more_ or _less_ than is expected of them, based on the size of their team. Represented as a percentage (0-100%).
+
+In the above dataset, the player's contribution gap for the `#big-bees` project is equal to 13.34% ( 46.67% - 33.33% ).
+
+## Contribution Accuracy and Bias
+
+Contribution accuracy and bias reflect how well you estimate your contribution to projects.
+
+To demonstrate how these stats apply, we'll use a modified version of the above dataset, aggregating the team estimates into their average (mean):
+
+| Project      | Self estimate | Team estimate (avg) |
+|:-------------|:--------------|:--------------------|
+| #big-bees    | 50%           | 45%                 |
+| #red-rabbits | 28%           | 30%                 |
+| #tiny-tigers | 20%           | 30%                 |
+
+### Contribution Accuracy
+
+Contribution accuracy reflects how accurate your estimations are relative to the consensus. Represented as a percentage (0-100%).
+
+Zero is equal to perfect accuracy (i.e. you estimate your contribution the same as your peers do). The closer this stat is to zero, the more accurate your contribution estimates are.
+
+Your contribution accuracy _per project_ is the difference between self- and team-estimated contribution as an absolute number. So the contribution accuracy stats for the above projects would be:
+
+| Project      | Accuracy             |
 |:-------------|:---------------------|
 | #tiny-tigers | abs(20% - 30%) = 10% |
 | #big-bees    | abs(50% - 45%) = 5%  |
 | #red-rabbits | abs(28% - 30%) = 2%  |
 
-Based on these projects, your overall **discernment stat** would be the mean of (10%, 5%, 2%), or **5.67%**.
+Based on these projects, the _overall contribution accuracy stat_ is 5.67% (mean of [10%, 5%, 2%]).
+
+## Contribution Bias
+
+Contribution bias reflects whether you tend to over- or under- estimate your contribution. Represented as a percentage (0-100%).
+
+It is calculated the same as contribution accuracy, except that it does turn the difference into an absolute number.
+
+Your contribution bias _per project_ is the difference between self- and team-estimated contribution. So the contribution bias stats for the above projects would be:
+
+| Project      | Bias             |
+|:-------------|:-----------------|
+| #tiny-tigers | 20% - 30% = -10% |
+| #big-bees    | 50% - 45% = 5%   |
+| #red-rabbits | 28% - 30% = -2%  |
+
+Based on these projects, the _overall contribution bias stat_ is -2.33% (mean of [-10%, 5%, -2%]).
```

Comment:
Tell me what i means if the number is negative or positive, and that I should aim for zero.